sql/core/src/test/scala/org/apache/spark/sql/execution/command/DescribeTableParserSuite.scala

```diff
@@ -75,6 +76,18 @@ class DescribeTableParserSuite extends AnalysisTest {
         UnresolvedAttribute(Seq("col")),
         isExtended = true))
 
+    val error = intercept[AnalysisException] {
+      comparePlans(parsePlan("DESCRIBE EXTENDED t col AS JSON"),
```

Comment:
since we are checking the error, we can just do `intercept[AnalysisException](parsePlan("DESCRIBE EXTENDED t col AS JSON"))`